etor/src/main/java/gov/hhs/cdc/trustedintermediary/external/database/DatabasePartnerMetadataStorage.java

```diff
@@ -38,18 +42,66 @@ public Optional<PartnerMetadata> readMetadata(final String uniqueId)
     @Override
     public void saveMetadata(final PartnerMetadata metadata) throws PartnerMetadataException {
         logger.logInfo("saving the metadata");
+        List<DbColumn> values = new ArrayList<>();
+
+        DbColumn dbColumn1 =
+                new DbColumn(
+                        "received_message_id",
+                        metadata.receivedSubmissionId(),
+                        false,
+                        Types.VARCHAR);
+        values.add(dbColumn1);
+
+        DbColumn dbColumn2 =
```

Comment:
Could you refactor this to only use one dbColumn? Just reinitialize and add the column at each step. Then we don't have to have 11+ different columns

Reply:
Definitely.

Reply:
Could this be another option to make it more concise and easier to maintain:
```
List<DbColumn> values = Stream.of(
        new DbColumn("received_message_id", metadata.receivedSubmissionId(), false, Types.VARCHAR),
        new DbColumn("sent_message_id", metadata.sentSubmissionId(), true, Types.VARCHAR),
        new DbColumn("sender", metadata.sender(), false, Types.VARCHAR),
        ...
).collect(Collectors.toList());
```

Reply:
@basiliskus, I think the only downside is that some of the `DbColumn`s are conditionally created with their value.  See `time_received`.  So, I could create all the conditional values up front and then do the `Stream.of(` call.  But then the creation of the conditional value is create farther away from where it is used.  Thoughts?

Reply:
This could be a good use case for ternary inline statements:
`new DbColumn("time_received", metadata.timeReceived() != null ? Timestamp.from(metadata.timeReceived()) : null, false, Types.TIMESTAMP),`

Reply:
Oh, right!

Reply:
Done.